tests/queries/0_stateless/02415_all_new_functions_must_be_documented.reference

```diff
@@ -130,6 +130,7 @@ arrayReverse
 arrayReverseFill
 arrayReverseSort
 arrayReverseSplit
+arrayShuffle
```

Comment:
Simply copy-paste the markdown docs content to the documentation object inside the registerFunction.

The `asin.cpp` as an example.